### PR TITLE
Сreated mock data component

### PR DESCRIPTION
**Pull request by alex6061**

### CODE REVIEW

**Comment by alex6061** [comment]
I also added pictures to the project

File: assets/img/download_image.svg
Comment (nagorapro) [inline assets/img/download_image.svg:1]: Imagine that this is a photograph and not a drawing
Look at the corners of this photo, it is not loaded correctly
In the design, these photos are presented separately for upload
Use jpg format
Use name 2.jpg

File: assets/img/warranty_image.svg
Comment (nagorapro) [inline assets/img/warranty_image.svg:1]: Imagine that this is a photograph and not a drawing
Look at the corners of this photo, it is not loaded correctly
In the design, these photos are presented separately for upload
Use jpg format
Use name 3.jpg

File: js/components/widgets/download/data/Data.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      description: 'link to apple store',
+    },
+  },
+  image:{
+    src: './../../../../../assets/img/download_image.svg',
+    description: 'image for download section',
+  }
+  }, 
+}
+
```
Comment (nagorapro) [inline js/components/widgets/download/data/Data.js:24]: BAD!!! :)

**Review by nagorapro — COMMENTED** [review]
Need fix

File: js/components/widgets/download/data/Data.js [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
-    priority: 1,
-  },
-  text:'See how we can help with making your shopping experience and money management more ZEN.',
-  links:{
-    google_link:{
-      src:'https://play.google.com',
-      description: 'link to google play',
+  download: {
+    title: {
+      content: "Peaceful shopping mindful money",
```
Comment (nagorapro) [inline js/components/widgets/download/data/Data.js:4]: Use single quotes '...'

File: js/components/widgets/download/data/Data.js [hunk 3]
```diff
@@ -1,24 +1,23 @@
...
+      priority: 1,
+    },
+    text: "See how we can help with making your shopping experience and money management more ZEN.",
+    links: {
+      google_link: {
```
Comment (nagorapro) [inline js/components/widgets/download/data/Data.js:9]: Use camelCase style for naming
gooleLink or google

File: js/components/widgets/download/data/Data.js [hunk 4]
```diff
@@ -1,24 +1,23 @@
...
-  links:{
-    google_link:{
-      src:'https://play.google.com',
-      description: 'link to google play',
+  download: {
+    title: {
+      content: "Peaceful shopping mindful money",
+      priority: 1,
+    },
+    text: "See how we can help with making your shopping experience and money management more ZEN.",
```
Comment (nagorapro) [inline js/components/widgets/download/data/Data.js:7]: Use property texts: ['Some text'],

File: js/components/widgets/download/data/Data.js [hunk 5]
```diff
@@ -1,24 +1,23 @@
...
+    links: {
+      google_link: {
+        src: "https://play.google.com",
```
Comment (nagorapro) [inline js/components/widgets/download/data/Data.js:10]: Use property source: 'https://...'

File: js/components/widgets/download/data/Data.js [hunk 6]
```diff
@@ -1,24 +1,23 @@
...
+      apple_link: {
+        src: "https://www.apple.com/store",
+        description: "link to apple store",
+      },
     },
-    apple_link:{
-      src:'https://www.apple.com/store',
-      description: 'link to apple store',
+    image: {
+      src: "./../../../../../assets/img/download_image.svg",
```
Comment (nagorapro) [inline js/components/widgets/download/data/Data.js:19]: source: 'assets/img/1.jpg',

**Review by nagorapro — COMMENTED** [review]
Need fix

Path to data.js file: js/data/data.js

**Review by nagorapro — COMMENTED** [review]
Need fix
Use the git flow commit message
Not "Fix"
"Fixed bla bla bla bla"

File: js/data/data.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    texts: [
+      'See how we can help with making your shopping experience and money management more ZEN.',
+    ],
+    links: {
+      googleLink: {
+        src: 'https://play.google.com',
+        description: 'google',
+      },
+      appleLink: {
+        src: 'https://www.apple.com/store',
```
Comment (nagorapro) [inline js/data/data.js:16]: I recommend "url" for name

File: js/data/data.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+    title: {
+      content: 'Peaceful shopping mindful money',
+      priority: 1,
+    },
+    texts: [
+      'See how we can help with making your shopping experience and money management more ZEN.',
+    ],
+    links: {
+      googleLink: {
+        src: 'https://play.google.com',
```
Comment (nagorapro) [inline js/data/data.js:12]: I recommend "url" for name

**Review by nagorapro — COMMENTED** [review]
Need fix

**Review by nagorapro — APPROVED** [review]
Done
Edit the commit history

**Pull request merged**